### PR TITLE
Enable profile tap on touch devices

### LINKED ISSUE #164: Touch a face doesn't work on mobile
When you double tap a face on a mobile you dont get the same behavior as if you double clicked with a mouse on a desktop

### PR DESCRIPTION
**Pull request by geoffwhittington**
Fixes #164

### CODE REVIEW

**Comment by hhaidar** [comment]
Can you also make @names work on single click/touch?

**Comment by geoffwhittington** [comment]
I don't follow what you mean here, sorry.

**Comment by hhaidar** [comment]
The text beside the avatars should be clickable too

**Comment by geoffwhittington** [comment]
@hhaidar Can you take a look - was this what you meant?

**Comment by sibartlett** [comment]
This change doesn't just enable touch, it fires on all touch events including swiping and dragging.

If you scroll the messages or users - you can end up inserts mentions into the text field.

File: media/js/views/room.js [hunk 1]
```diff
@@ -58,7 +58,8 @@
             'click .hide-edit-room': 'hideEditRoom',
             'click .submit-edit-room': 'submitEditRoom',
             'click .archive-room': 'archiveRoom',
-            'dblclick .lcb-avatar': 'poke'
+            'dblclick .lcb-avatar, .lcb-message-name, .lcb-room-sidebar-user-name, .lcb-room-sidebar-user-username': 'poke',
```
Comment (hhaidar) [inline media/js/views/room.js]: Should be single click I think.

**Comment by hhaidar** [comment]
I updated some stuff. If it's good let's merge it in.

**Comment by sibartlett** [comment]
Ideally, if you're on mobile and you tap an user in the sidebar - the sidebar should close before focusing the textbox.

Also, on iOS when the text box is focused, the cursor sits before the text and not after it.

**Comment by hhaidar** [comment]
@sibartlett do you mind creating tickets for the ios issues?

**Pull request merged**